src/main/java/com/iridium/iridiumskyblock/managers/SchematicManager.java

```diff
@@ -100,17 +101,17 @@ private void pasteSchematic(final Island island, final World world, final Schema
         // Loop all blocks in the schematics layer at the current y level
         for (int x = 0; x < schematic.getLength(); x++) {
             for (int z = 0; z < schematic.getWidth(); z++) {
-                Block block = island.getCenter(world).subtract(schematic.getLength()/2.00, -90, schematic.getWidth()/2.00).add(x, y, z).getBlock();
+                Block block = island.getCenter(world).subtract(schematic.getLength()/2.00, 0, schematic.getWidth()/2.00).add(x, yHeightIsland + y, z).getBlock();
```

Comment:
Not an issue with your code but let's change it :p
```suggestion
                Block block = island.getCenter(world).subtract(schematic.getLength() / 2.0, 0, schematic.getWidth() / 2.0).add(x, yHeightIsland + y, z).getBlock();
```